src/main/java/uk/gov/hmcts/reform/em/stitching/pdf/PDFWatermark.java

```diff
@@ -0,0 +1,67 @@
+package uk.gov.hmcts.reform.em.stitching.pdf;
+
+import org.apache.pdfbox.multipdf.Overlay;
+import org.apache.pdfbox.pdmodel.PDDocument;
+import org.apache.pdfbox.pdmodel.PDPage;
+import org.apache.pdfbox.pdmodel.PDPageContentStream;
+import org.apache.pdfbox.pdmodel.common.PDRectangle;
+import org.apache.pdfbox.pdmodel.graphics.image.PDImageXObject;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.em.stitching.domain.BundleDocument;
+import uk.gov.hmcts.reform.em.stitching.domain.DocumentImage;
+import uk.gov.hmcts.reform.em.stitching.domain.enumeration.ImageRenderingLocation;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+@Service
+public class PDFWatermark {
+
+    public void processDocumentWatermark(File documentImageFile,
+                                                              Map<BundleDocument, File> documents,
+                                                              DocumentImage documentImage) throws IOException {
```

Comment:
formatting?

Reply:
Not sure what happened there. Resolved in next commit